easybuild/easyconfigs/f/FSON/FSON-1.0.5-GCCcore-8.3.0.eb

```
@@ -0,0 +1,27 @@
+easyblock = "CMakeMake"
+
+name = 'FSON'
+version = '1.0.5'
+
+homepage = "https://github.com/josephalevin/fson"
+description = "Fortran 95 JSON Parser"
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['https://github.com/josephalevin/fson/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['d52f928d59b090eb47ed2bc880ee58d46cd8dd7e554e3df03c7c517d7a6b6012']
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('CMake', '3.15.3'),
+]
+
+configopts = "-DBUILD_SHARED_LIBS=ON"
```

```suggestion
configopts = ['-DBUILD_SHARED_LIBS=OFF', '-DBUILD_SHARED_LIBS=ON']
```
usually nice to provide both (though this is not a requirement)